src/sparseml/pytorch/optim/quantization/helpers.py

```diff
@@ -37,15 +39,40 @@
 ]
 
 
+_QUANTIZABLE_MODULE_TYPES = (
+    {
+        # Conv based layers
+        torch.nn.Conv1d,
+        torch.nn.Conv2d,
+        torch.nn.Conv3d,
+        nni.ConvBn1d,
+        nni.ConvBn2d,
+        nni.ConvBn3d,
+        nni.ConvReLU1d,
+        nni.ConvReLU2d,
+        nni.ConvReLU3d,
+        nni.ConvBnReLU1d,
+        nni.ConvBnReLU2d,
+        nni.ConvBnReLU3d,
+        # Linear Layers
+        torch.nn.Linear,
+        nni.LinearReLU,
+    }
+    if nni
+    else None
```

Comment:
Better to be [] instead of None? (see below)

Reply:
see other comment